tests/_data/manifests/src/integrator-manifests-master/Spryker/TestIntegratorConfigureModule/1.0.3/installer-manifest.json

```diff
@@ -1,8 +1,161 @@
+                }
+            }
+        },
+        {
+            "target": "\\Pyz\\Zed\\TestIntegratorDefault\\TestIntegratorDefaultConfig::testChange5",
+            "value": {
+                "value": "array_merge(parent::isCartCartItemsViaAjaxLoadEnabled(), parent::getSharedConfig(), $this->getSharedConfig2())",
+                "is_literal": true
+            }
+        },
+        {
+            "target": "\\Pyz\\Zed\\TestIntegratorDefault\\TestIntegratorDefaultConfig::testChange7",
+            "value": [
+                "\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_CHANGE7"
+            ]
+        },
+        {
+            "target": "\\Pyz\\Zed\\TestIntegratorDefault\\TestIntegratorDefaultConfig::testChange8",
+            "value": {
+                "\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::NEW_VALUE": [
+                    "\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_NEW_VALUE"
+                ]
+            }
+        },
+        {
+            "previousValue": "$array = parent::isCartCartItemsViaAjaxLoadEnabledChanged();\n        $array = array_merge_recursive(\n            $array,\n            [\n                \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE_NOT_CHANGED,\n                \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE2_NOT_CHANGED\n            ]\n        );\n\n        return array_merge(\n            $array,\n            [\n                \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE_NOT_CHANGED => [\n                    \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::MANIFEST_KEY_NOT_CHANGED\n                ],\n                \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE2_NOT_CHANGED => [\n                    \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::MANIFEST_KEY22_NOT_CHANGED\n                ]\n            ]\n        );",
+            "target": "\\Pyz\\Zed\\TestIntegratorDefault\\TestIntegratorDefaultConfig::testChange6",
+            "value": {
+                "value": "$array = parent::isCartCartItemsViaAjaxLoadEnabledChanged(); $array = array_merge($array, [\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE, \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE2]); array_merge($array, [\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE => [\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::MANIFEST_KEY], \\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::TEST_VALUE2 => [\\App\\Manifest\\Generator\\ArrayConfigElementManifestStrategy::MANIFEST_KEY22]])",
+                "is_literal": true
+            }
+        },
+        {
+            "previousValue": {
+                "static::IS_CART_CART_ITEMS_VIA_AJAX_LOAD_ENABLED": "false"
```

Comment:
this syntax (I mean object inside of previousValue) must be documented. 
Also for me it's not clear why we are using string instead of booleans here?

Reply:
Fixed for boolean. It's related only for literal value because if we have simple expression we only append new constants/values and not checked previous, for literal value it's already described